files/docker-compose.yml

```diff
@@ -10,12 +10,12 @@ secrets:
 services:
   update:
     command:
-      - "--cc=ncats-dev@beta.dhs.gov,ncats@hq.dhs.gov"
+      - "--cc=cisa-cyhy-mailer@gwe.cisa.dhs.gov,ncats@hq.dhs.gov"
```

Comment:
Why not use the GWE dev distribution?

Also, there is probably an updated email that replaces `ncats@hq.dhs.gov` as well.  @climber-girl probably knows the new email.

Reply:
`cisa-cyhy-mailer` is a group that we planned to use to receive `cyhy-mailer` summary and related emails (e.g. bounce notifications).  Our reasoning was that although these emails currently go to our dev team, there may come a day when other folks should get these emails as well.  Using a separate distro here gives us that flexibility without requiring a code change.